auth/src/api/mod.rs

```diff
@@ -0,0 +1,15 @@
+use std::net::SocketAddr;
+
+use axum::{Router, Server};
+
+mod builder;
+mod handlers;
+
+pub use builder::ApiBuilder;
+
+pub async fn serve(router: Router, address: SocketAddr) {
+    Server::bind(&address)
+        .serve(router.into_make_service())
+        .await
+        .unwrap_or_else(|_| panic!("Failed to bind to address: {}", address));
+}
```

Comment:
Because of how small this function is, I'm wondering if the refactor into its own file is worth it :man_shrugging:

If we want to reuse it, maybe it should be in common and gateway and deployer made to call it?

Reply:
Yeah, I agree, I'll look into refactoring this.

Reply:
I made it a method on ApiBuilder for now, like is done in gateway.